packages/core/src/client/index.ts

```diff
@@ -489,19 +493,109 @@ export class Web3InboxClient {
     ) => {
       const unsub = subscribe(data, () => {
         onNotificationDataUpdate(data);
-      })
+      });
 
       return {
         stopWatchingNotifications: () => {
-	  this.notifyClient.off("notify_message", notifyMessageListener)
-	  unsub();
-	},
+          this.notifyClient.off("notify_message", notifyMessageListener);
+          unsub();
+        },
         data,
         nextPage,
       };
     };
   }
 
+  /**
+   * Mark specified notifications as read for a account & domain subscription
+   * @param {string[]} notificationIds - Account to get subscription message history for, defaulted to current account
+   * @param {string} [account] - Account to get subscription message history for, defaulted to current account
+   * @param {string} [domain] - Domain to get subscription message history for, defaulted to one set in init.
+   */
+  public markNotificationsAsRead(
```

Comment:
This doesn't match the docs: https://github.com/WalletConnect/walletconnect-docs/pull/1376/files#diff-80e3d9506b02da954f3179df551346c8490cf64746729aa1f0526f3adb101a6aR420

Also we should use structs instead of parameter lists to make it easier to add parameters in the future in a non-breaking way.

Reply:
We should update the docs to match this, as it matches the rest of the SDK. All other functions use normal (non struct type) pasrams

Reply:
I'd be ok if we updated the docs instead. But I would prefer if we did the structs now for all new methods and parameters so it's less of a breaking change later when [everything uses struct parameters](https://github.com/WalletConnect/web3inbox-client/issues/70).

Also looks like the `unreadFirst` flag is non-existant here but it is in the docs.